.github/workflows/release.yml

```diff
@@ -0,0 +1,79 @@
+name: Build and publish AppImages
+
+env:
+  UBUNTU_RELEASE: jammy
+  UBUNTU_PUBKEY: 871920D1991BC93C
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        platform:
+          - linux/amd64
+          - linux/arm64
+    steps:
+      - uses: actions/checkout@v4
+      - uses: docker/setup-qemu-action@v2
+      - uses: docker/setup-buildx-action@v2
+      - uses: docker/build-push-action@v4
+        with:
+          platforms: ${{ matrix.platform }}
+          outputs: build
+          build-args: |
+            UBUNTU_RELEASE
+      - uses: rlespinasse/github-slug-action@v4
+      - name: Prepare AppImage build
+        env:
+          TARGET_PLATFORM: ${{ matrix.platform }}
+        shell: bash
+        run: |
+          set -eu
+          find build -mindepth 1 -maxdepth 1 -type d | while read -r path; do
+            icon_path="${path}/usr/share/icons/hicolor/scalable/mimetypes"
+            mkdir -p "${icon_path}"
+            touch "${icon_path}/no-icon.svg"
+
+            set -a
+            APPIMAGE_PATH="${path}"
+            APPIMAGE_VERSION="${GITHUB_REF_SLUG}"
+            APPIMAGE_APT_ARCH="${TARGET_PLATFORM#*/}"
+            APPIMAGE_APT_DISTRO="${UBUNTU_RELEASE}"
+            APPIMAGE_APT_PUBKEY="${UBUNTU_PUBKEY}"
+            APPIMAGE_ARCH="$(basename "${path}")"
+            printenv | grep ^APPIMAGE_ >>"${GITHUB_ENV}"
+          done
+      - uses: AppImageCrafters/build-appimage@v1
+      - uses: actions/upload-artifact@v3
+        with:
+          name: appimages
+          path: ./*.AppImage
```

Comment:
You need to include `*.AppImage.zsync` files, they are currently missing from CI output.

Reply:
Done.